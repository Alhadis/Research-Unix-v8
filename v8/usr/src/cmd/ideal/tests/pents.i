pents
.nf
.IS
...degrees
box pentagon
{
	var pt1, pt2, pt3, pt4, pt5;
	var radius, center;
	pt1 = center + radius;
	pt2 = center + cis (72)*radius;
	pt3 = center + cis (144)*radius;
	pt4 = center + cis (-144)*radius;
	pt5 = center + cis (-72)*radius;
	conn pt1 to pt2;
	conn pt2 to pt3;
	conn pt3 to pt4;
	conn pt4 to pt5;
	conn pt5 to pt1;
	bdlist = 1.1[center, pt1],
		1.1[center, pt2],
		1.1[center, pt3],
		1.1[center, pt4],
		1.1[center, pt5]
	;
}
main
{
	put P1:pentagon{center = (1,0); radius = (0,0.25);};
	put P2:pentagon{pt3 = P1.pt4; radius = (0,0.5);opaque;};
	put P3:pentagon{pt3 = P2.pt4; radius = (0,0.75);opaque;};
	put P4:pentagon{pt3 = P3.pt4; radius = (0,1.0);opaque;};
}
.IE
.IS
...forget pentagon
.IE
