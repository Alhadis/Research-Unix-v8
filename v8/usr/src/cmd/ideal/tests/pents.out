pents
.nf
.IS
...maxx 4.15525
...maxy 1.60676
...minx 0.762236
...miny -0.202254
...line 3.20419 1.60676 4.15525 0.91578
...line 4.15525 0.91578 3.79198 -0.202254
...line 3.79198 -0.202254 2.61641 -0.202254
...line 2.61641 -0.202254 2.25314 0.91578
...line 2.25314 0.91578 3.20419 1.60676
...line 2.17557 1.15451 2.30983 1.05697
...line 2.53134 -0.202254 1.73473 -0.202254
...line 1.73473 -0.202254 1.46228 0.636271
...line 1.46228 0.636271 2.17557 1.15451
...line 1.44084 0.702254 1.44535 0.698975
...line 1.67093 -0.202254 1.14695 -0.202254
...line 1.14695 -0.202254 0.965311 0.356763
...line 0.965311 0.356763 1.44084 0.702254
...line 1.10441 -0.202254 0.853054 -0.202254
...line 0.853054 -0.202254 0.762236 0.0772542
...line 0.762236 0.0772542 0.96559 0.225
.IE
.IS
.IE
